client/src/Movies/SavedList.js

```diff
@@ -1,4 +1,5 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import { Link, NavLink } from "react-router-dom";
 
 export default class SavedList extends Component {
   constructor(props) {
```

Comment:
I know this was given to you initially, but as we change things & add our own code we should want to modify the rest of the code to fit our changes